conf/application.conf

```diff
@@ -0,0 +1,400 @@
+
+# The application languages
+# ~~~~~
+application.langs = "en"
+
+# Global object class
+# ~~~~~
+# Define the Global object class for this application.
+# Default to Global in the root package.
+application.global= uk.gov.hmrc.helptosaveproxy.config.MicroserviceGlobal
+
+# Router
+# ~~~~~
+# Define the Router object to use for this application.
+# This router will be looked up first when the application is starting up,
+# so make sure this is the entry point.
+# Furthermore, it's assumed your route file is named properly.
+# So for an application router like `my.application.Router`,
+# you may need to define a router file `conf/my.application.routes`.
+# Default to Routes in the root package (and conf/routes)
+# !!!WARNING!!! DO NOT CHANGE THIS ROUTER
+application.router = prod.Routes
+
+
+# Evolutions
+# ~~~~~
+# You can disable evolutions if needed
+# evolutionplugin=disabled
+
+# Logger
+# ~~~~~
+# You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
+
+# Root logger:
+logger.root = ERROR
+
+# Logger used by the framework:
+logger.play = INFO
+
+# Logger provided to your application:
+logger.application = DEBUG
+
+# Metrics plugin settings - graphite reporting is configured on a per env basis
+metrics {
+  name = ${appName}
+  rateUnit = SECONDS
+  durationUnit = SECONDS
+  showSamples = true
+  jvm = true
+  enabled = true
+}
+
+
+# Microservice specific config
+
+auditing {
+  enabled = true
+  traceRequests = true
+  consumer {
+    baseUri {
+      host = localhost
+      port = 8100
+    }
+  }
+}
+
+
+mongodb {
+  uri = "mongodb://localhost:27017/help-to-save"
+}
+
+microservice {
+  metrics {
+    graphite {
+      host = graphite
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
+    }
+  }
+
+  services {
+
+    auth {
+      host = localhost
+      port = 8500
+    }
+
+    nsi {
+      host = localhost
+      port = 7002
+      manage-account {
+        url = "https://nsandi.com"
+      }
+      client {
+        base64KeystoreType = "jks"
+        base64KeystorePassword = ""
+        base64Keystore = ""
+
+        phase2{
+          set1 {
+            base64KeystorePassword = ""
+            base64Keystore = ""
+          }
+          set2 {
+            base64KeystorePassword = ""
+            base64Keystore = ""
+          }
+        }
+
+        httpheader {
+          basicauth {
+            Base64User = "dXNlcm5hbWU=" //username
+            Base64Password = "cGFzc3dvcmQ=" //password
+          }
+          header-key = "Authorization-test"
+          encoding = "UTF-8"
+        }
+      }
+      server {
+        base64Certificate = ""
+        base64TruststorePassword = ""
+
+        phase2{
+          set1 {
+            base64Certificate = ""
+            base64TruststorePassword = ""
+          }
+          set2 {
+            base64Certificate = ""
+            base64TruststorePassword = ""
+          }
+        }
```

Comment:
we can remove `phase2` config here as it never works

Reply:
ok its gone!